main.go

```diff
@@ -832,7 +875,11 @@ func (p *gateway) start(ctx context.Context) error {
 	// setup scheduler
 	scheduler := p.setupScheduler(p.config)
 
-	if err := p.setupDebugServer(p.config, p.logger, scheduler); err != nil {
+	// set up defaultDelayScheduler
+	defaultDelayScheduler := p.setupDefaultDelayScheduler(p.config)
+	defaultDelayScheduler.Prefix = gatewayMetricsPrefix
```

Comment:
this can be set in the same function above as well. we don't need 3 places to configure a scheduler.